tests/factories.py

```diff
@@ -117,7 +117,7 @@ class Meta:
         sqlalchemy_session_persistence = "commit"
 
 
-class ReservationFactory(factory.alchemy.SQLAlchemyModelFactory):
+class ReservationFactory(DictMixin, factory.alchemy.SQLAlchemyModelFactory):
```

Comment:
@vladarefiev what is ``DictMixin`` for?

Reply:
@amihaiemil to get a object converted to dict